### PR TITLE
Added exporting possibility

### PR DESCRIPTION
**Pull request by Oustad**
Hi, 

I added an extra module which supports the highcharts exporting feature.
Also added a demo of it in the demo folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-    "react-highlight": "^0.4.1",
-    "webpack": "^1.5.3",
-    "webpack-dev-server": "^1.10.1"
-  }
+    "name": "react-highcharts",
+    "version": "2.0.0",
+    "description": "React wrapper for highcharts",
+    "main": "index.js",
+    "scripts": {
+        "test": "webpack && mocha",
```
Comment (kirjs) [inline package.json]: You should be able to just say webpack-dev-server

File: package.json [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
-    "imports-loader": "^0.6.4",
-    "jsdom": "^6.3.0",
-    "mocha": "^2.3.0",
-    "raw-loader": "^0.5.1",
-    "react": "*",
-    "react-highlight": "^0.4.1",
-    "webpack": "^1.5.3",
-    "webpack-dev-server": "^1.10.1"
-  }
+    "name": "react-highcharts",
```
Comment (kirjs) [inline package.json]: Please undo reformatting the whole file

File: webpack.config.js [hunk 3]
```diff
@@ -1,47 +1,45 @@
...
-      }
-    }
-  ],
-  resolve: {
-    alias: {
-      "highstock" : "highstock-release/highstock.src.js",
-      "highcharts" : "highcharts-release/highcharts.src.js",
-      "highcharts-more" : "highcharts-release/highcharts-more.src.js",
-      "highcharts-standalone-adapter" : "highcharts-release/adapters/standalone-framework.src.js"
+    module: {
```
Comment (kirjs) [inline webpack.config.js]: same thing with indentation as above, let's keep 2 spaces

**Comment by kirjs** [comment]
Left you several comments, mostly formatting, ptal

**Comment by Oustad** [comment]
Hi, thanks for the feedback, added a new commit now. Reformatted the files, and removed the local path I had snuck in.

File: package.json [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
-    "highstock-release": "^2.1.8",
-    "imports-loader": "^0.6.4",
-    "jsdom": "^6.3.0",
-    "mocha": "^2.3.0",
-    "raw-loader": "^0.5.1",
-    "react": "*",
-    "react-highlight": "^0.4.1",
-    "webpack": "^1.5.3",
-    "webpack-dev-server": "^1.10.1"
+  "devDependencies":{
```
Comment (kirjs) [inline package.json]: Now there is a space after `:`.
I don't think this file was changed at all.

File: src/Highcharts.jsx [hunk 5]
```diff
@@ -1,4 +1,4 @@
 global.HighchartsAdapter = require('exports?HighchartsAdapter!highcharts-standalone-adapter');
 var Highcharts = require("exports?Highcharts!highcharts");
 var chartFactory = require('./chartsFactory.jsx');
-module.exports = chartFactory(Highcharts, 'Chart');
\ No newline at end of file
+module.exports = chartFactory(Highcharts, 'Chart');
```
Comment (kirjs) [inline src/Highcharts.jsx]: Was this file changed at all?

**Comment by kirjs** [comment]
Thanks, it looks better, but there are still some formatting issues, can you please make sure that added/removed spaces and changed indentation won't be reflected in the diff?

It makes the PR twice larger than it really is, and it takes some extra effort to find the real changes.

**Comment by kirjs** [comment]
I run the demo, and it seems to work pretty well. 

My only question is why do we need to create ExportableHighcharts.jsx file?
Why wouldn't we just add exportable the demo code?

**Comment by kirjs** [comment]
Also it would be really nice to have a section in the readme

**Comment by Oustad** [comment]
Sorry about the formatting issues, have autoformat on save in atom and I use different formatting rules :)
I've added sections in readme now.
Yes, you could just do  ```require('exporting');``` in the regular highcharts example. I for my part think it gives a cleaner surface api to bring into another application with the ExportableHighcharts component, but that might just be my opinion.

**Comment by kirjs** [comment]
Thanks a lot, the code looks much better now!

Just 2 more things I'd like to clarify:
* We have `exporting` and `exportable` in the code, I'm trying to see a logic behind using different words, but can't quite get it. Could we maybe stick with 1 way of naming it everywhere?
* My only concern is that if you bundle highcharts with exportable, is that now you can't add exportable to highstocks, or to highcharts with more.js, because then you'd have 2 copies of highcharts bundled (e.g. one from highstocks and one from exporting). How would you suggest we handle this case?

**Comment by kirjs** [comment]
Fixed in #45

**Pull request closed without merging**